test/support/mediaSettingsTests.js

```diff
@@ -175,12 +181,11 @@ export function runSubtitlesTests() {
             cy.getByTestId('bp-media-settings-subtitles').contains('Off');
 
             // Toggling CC back on should restore the previously used text track
-            cy.getByTitle('Subtitles/Closed Captions')
-                .as('subtitlesBtn')
-                .should('be.visible')
-                .should('have.attr', 'aria-pressed', 'false')
-                .click()
-                .should('have.attr', 'aria-pressed', 'true');
+            cy.getByTitle('Subtitles/Closed Captions').as('subtitlesBtn');
```

Comment:
Minor, but we could alias this button using `.as('subtitlesBtn')` and then use `cy.get('@subtitlesBtn')` to stay consistent with the lines above.